scipy/linalg/lapack.py

```diff
@@ -736,7 +736,8 @@
     _clapack = None
 
 # Backward compatibility
-from .blas import find_best_blas_type as find_best_lapack_type
+# Can this be removed now?
+# from .blas import find_best_blas_type as find_best_lapack_type
```

Comment:
@pv @rgommers Does `find_best_lapack_type` still need to be preserved for backwards compatibility?  It looks as though that line went in a while ago(2012 IIRC).  It's not used in SciPy, all uses are of `find_best_blas_type`
Is `find_best_lapack_type` still needed for any non-obvious purpose?

Reply:
It's a public function, so we can't just remove it: http://scipy.github.io/devdocs/generated/scipy.linalg.find_best_blas_type.html

`find_best_lapack_type` is not public though, so that could be removed.

note, probably this distinction is an accident of history

Reply:
OK. I removed the 
`from .blas import find_best_blas_type as find_best_lapack_type` 
line as  `find_best_lapack_type` isn't used and isn't public.